### PR TITLE
Adding the Personal Access Token to Set-ConfluenceInfo 

### PR DESCRIPTION
**Pull request by mdg1-uo**
## Description

- Added the -PersonalAccessToken parameter to Set-Info and all module commands.
- Updated documentation for usage.

## Motivation and Context

The ability to use a Personal Access Token was added to the API, but not to the ConfluencePS module. This pull request will add this capability.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] I have added Pester Tests that describe what my changes should do.
- I was not sure exactly how to do the Pester Test.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/en-US/commands/Set-Info.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Specify wiki location and authorization for use in this session's REST API reque
 ## SYNTAX
 
 ```powershell
-Set-ConfluenceInfo [[-BaseURi] <Uri>] [[-Credential] <PSCredential>] [[-PageSize] <UInt32>] [-PromptCredentials]
+Set-ConfluenceInfo [[-BaseURi] <Uri>] [[-Credential] <PSCredential>] [-PersonalAccessToken <PersonalAccessToken>] [[-PageSize] <UInt32>] [-PromptCredentials]
```
Comment (lipkau) [inline docs/en-US/commands/Set-Info.md:19]: You defined PersonalAccessToken as type `String`

**Review by lipkau — CHANGES_REQUESTED** [review]
You did not update the documentation of all the other public functions where you have added the new parameter.

Personally I would have a secret (as the PAT is) as a `SecureString`. But that would make it very annoying to use. So I am fine with PersonalAccessToken being a `string`

**Comment by mdg1-uo** [comment]
> You did not update the documentation of all the other public functions where you have added the new parameter.
> 
> Personally I would have a secret (as the PAT is) as a `SecureString`. But that would make it very annoying to use. So I am fine with PersonalAccessToken being a `string`

I thought about typing this as a `SecureString` which would be consistent with PowerShell 6+ Invoke-WebRequest `Token` parameter, but I noticed that in the private Invoke-WebRequest, the newer Authentication parameters weren't being used completely so I did take the easier road. 

I can work on that in a future change though.

**Review by lipkau — APPROVED** [review]

**Comment by CoolID2** [comment]
Hello, 
what is the status here? 
I need to update a Confluence page via PS and it would be great if I could use the PersonalAccessToken from this module.

**Comment by mdg1-uo** [comment]
> Hello, what is the status here? I need to update a Confluence page via PS and it would be great if I could use the PersonalAccessToken from this module.

It's been merged to the master branch so you should be able to download the ConfluencePS directory into your PSModulePath directory. I'm not sure what the process is for them to update the PSGallery version though.

**Comment by ZonderP** [comment]
Is this needed this way?
I just had to change a project of mine which creates Confluence pages with ConfluencePS since our company changed from on-pre Confluence to cloud Confluence where we now need to use access tokens.
Is a 'Personal Access Token' something different?
Because for me just using the Credential as supported so far by ConfluencePS works well with a Credential with a user name and the 'Access Token' string as password.

**Pull request merged**